.ahoy/site/.ahoy.yml

```diff
@@ -5,6 +5,18 @@ commands:
     usage: Run drush via ahoy (helpful for abstraction)
     cmd: ahoy cmd-proxy drush --root=docroot "{{args}}"
 
+  uli:
```

Comment:
We need to move this back to `ahoy/dkan.ahoy.yml` because dkan_starter uses it but dkan_starter has it's own ahoy.yml OR we need to duplicate this in dkan_starter's own .ahoy.yml file.